src/main/java/address/MainApp.java

```diff
@@ -61,6 +63,10 @@ protected void setupComponents() {
         storageManager = new StorageManager(modelManager);
         mainController = new MainController(this, modelManager, config);
         syncManager = new SyncManager();
+
+        shortcutsManager = new ShortcutsManager();
+        EventManager.getInstance().registerHandler(shortcutsManager);
```

Comment:
Should this be placed in the constructor of `ShortcutsManager` instead? To be consistent with the other components which involve event handlers.